Source/drlg_l2.cpp

```diff
@@ -2921,6 +2904,7 @@ void FixDoors()
 void GenerateLevel(lvl_entry entry)
 {
 	bool doneflag = false;
+	uint8_t floorID = 3;

```

Comment:
```suggestion
	constexpr uint8_t CatacombsFloorID = 3;
```
Same situation as the previous comment :)